components/forms/w-input.vue

```diff
@@ -1,39 +1,54 @@
 <template>
-  <w-field v-bind="{ ...$attrs, ...$props }" #default="{ triggerValidation, aria }" :class="{ 'has-suffix': $slots.suffix, 'has-prefix': $slots.prefix }">
-    <div class="input" :class="inputWrapperClass">
+  <w-field v-bind="{ ...$attrs, ...$props }" #default="{ triggerValidation, aria }">
+    <div class="relative">
```

Comment:
same question here about moving this to component-classes

Reply:
Yes, let's do it